### PR TITLE
[pt-br] add content/pt-br/docs/tasks/configure-pod-container/assign-p…

### PR DESCRIPTION
**Pull request by DonatoHorn**

### CODE REVIEW

File: content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    Na saída anterior, você pode ver que o nó `worker0` tem o rótulo `disktype=ssd`.
+
+## Crie um pod que é agendado em seu nó escolhido
+
+Este arquivo de configuração de pod descreve um pod que tem um seletor de nó, 
+`disktype: ssd`. Isto significa que o pod vai ser agendado em um nó que tem o rótulo `disktype=ssd`.
```
Comment (devlware) [inline content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md:67]:
```suggestion
`disktype: ssd`. Isto significa que o pod será agendado em um nó que tem o rótulo `disktype=ssd`.
```

File: content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+{{< codenew file="pods/pod-nginx.yaml" >}}
+
+1. Use o arquivo de configuração para criar um pod que é agendado em um nó escolhido:
```
Comment (devlware) [inline content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md:71]:
```suggestion
1. Use o arquivo de configuração para criar um pod que será agendado no nó escolhido:
```

File: content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    
+    ```shell
+    kubectl apply -f https://k8s.io/examples/pods/pod-nginx.yaml
+    ```
+
+1. Verifique se o pod é executando em seu nó escolhido:
```
Comment (devlware) [inline content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md:77]:
```suggestion
1. Verifique se o pod está executando no nó escolhido:
```

File: content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+    ```
+
+## Crie um pod que é agendado em um nó específico
+
+Você pode também agendar um pod para um nó específico usando `nodeName`.
+
+{{< codenew file="pods/pod-nginx-specific-node.yaml" >}}
+
+Use o arquivo de configuração para criar um pod que será agendado somente no `foo-node`.
```
Comment (devlware) [inline content/pt-br/docs/tasks/configure-pod-container/assign-pods-nodes.md:96]:
```suggestion
Use o arquivo de configuração para criar um pod que será agendado somente no nó `foo-node`.
```

File: content/pt-br/examples/pods/pod-nginx-specific-node.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  nodeName: foo-node # agende o Pod para um Nó específico
```
Comment (devlware) [inline content/pt-br/examples/pods/pod-nginx-specific-node.yaml:6]: You can keep the comments in english

**Review by devlware — DISMISSED** [review]

**Comment by DonatoHorn** [comment]
Hi, devlware, thanks for reviewing.

**Comment by devlware** [comment]
@stormqueen1990 would you like to stamp this too?

**Comment by stormqueen1990** [comment]
@devlware looks good to me!

**Pull request merged**